databricks/koalas/indexes.py

```diff
@@ -861,6 +861,24 @@ def is_object(self):
         """
         return is_object_dtype(self.dtype)
 
+    def is_type_compatible(self, kind):
```

Comment:
Can you add it to the docs?

Reply:
pandas doesn't add this function to their docs.

Shall we add this to the docs regardless of it or keep excluding ?

Reply:
They have the doc page https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Index.is_type_compatible.html but seems it's not just listed up. Can you add it with hidden option as below?

```rst
.. toctree::
   :hidden:
   Index.is_type_compatible
```

Reply:
Oh I didn't notice that hidden option. Thanks, I'll add it !! 👍